### PR TITLE
11709 v1 defaults to camel case

### PR DESCRIPTION
**Pull request by darkbushido**
## Description of change
The fast jsonapi gem supports camelCase. With this, the v1 endpoint won't need olive-branch to re-translate the json

## Original issue(s)
department-of-veterans-affairs/va.gov-team#11709

## Things to know about this PR
* set_key_transform :camel_lower

### CODE REVIEW

**Comment by LindseySaari** [comment]
Looks like some failing specs. I'll review once passing

File: spec/lib/lighthouse/facilities/client_spec.rb [hunk 1]
```diff
@@ -56,22 +57,24 @@
...
+      services: { 'health' => %w[Audiology Cardiology Dermatology Gastroenterology
+                                 MentalHealthCare Ophthalmology PrimaryCare SpecialtyCare],
+                  'last_updated' => '2021-01-25', 'other' => [] },
       feedback: {
         'effective_date' => nil,
         'health' => {}
       },
       access: {
-        'effective_date' => '2020-09-14',
+        'effective_date' => '2021-01-25',
```
Comment (LindseySaari) [inline spec/lib/lighthouse/facilities/client_spec.rb:68]: Does the effective_date attribute (and others that are still snake) need to be camel cased here?

Comment (darkbushido) [reply spec/lib/lighthouse/facilities/client_spec.rb:68]: Good catch, this is a bug in fast_jsonapi and a larger argument. let me see what I can do. [Deep transform keys for complex attributes](https://github.com/jsonapi-serializer/jsonapi-serializer/issues/62)

Comment (LindseySaari) [reply spec/lib/lighthouse/facilities/client_spec.rb:68]: Awesome 👍

**Comment by LindseySaari** [comment]
@darkbushido, it looks like we may be moving to jsonapi-serializer since fast jsonapi is no longer maintained. Just wanted to make you aware of this PR: https://github.com/department-of-veterans-affairs/vets-api/pull/5825/files

**Comment by darkbushido** [comment]
I was going to ask about that after this PR got merged. It looks like it's just a fork and you can see the complicated discussion on how to serialize JSON. I have to ask is an _ really this difficult to handle on the front end?

**Comment by LindseySaari** [comment]
> I was going to ask about that after this PR got merged. It looks like it's just a fork and you can see the complicated discussion on how to serialize JSON. I have to ask is an _ really this difficult to handle on the front end?

I mean, probably not, but the whole CamelCase/SnakeCase predates my time on the platform

File: spec/lib/lighthouse/facilities/client_spec.rb (hunk 1 above)
Comment (LindseySaari) [reply spec/lib/lighthouse/facilities/client_spec.rb:68]: I see the changes for deep_stringify_keys.deep_transform_keys in the serializer. Is this what you were referring to when you said that anything uploaded still need to be snake cased?

**Review by LindseySaari — APPROVED** [review]

**Pull request merged**